Contactmobile

david63011 opened the pull request:
I have uploaded my new mobile site with a working contact form

tufoinnkuo10 commented on index.html:
```diff
@@ -0,0 +1,346 @@
+                  rows="10"
+                  placeholder="Enter text here"
+                  required=""
+                ></textarea>
```
- [ ] great job, your form works and submits successfully. Let's work a little on the design to perfect it. Kindly remove the black border from the submit button
- [ ] please add the extra space below the contact form like in the Figma design
- [ ] kindly add the margin top/bottom of your inputs, and please make the placeholder text a little bit bigger 
your design 
![Screenshot (369)_LI](https://user-images.githubusercontent.com/65298048/187549229-434cf842-bc74-448f-bfe3-327b2811128b.jpg)
figma
![Screenshot (369)](https://user-images.githubusercontent.com/65298048/187549400-45bb34d2-8d5d-484b-baeb-47b8ea6510e5.png)

tufoinnkuo10 reviewed (changes requested):
Hi @david63011 

There are some issues that you still need to work on to go to the next project but you are almost there!

To Highlight
✅ You used the correct Github flow
✅ Descriptive pull request message
✅ linter error checks validated
✅ Documented your work professionally
✅form works and submits

Required Changes ♻️
Check the comments under the review
Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

david63011 commented:
Ok I fixed the border, the extra but I couldn't download the black indicator line I hope that is ok there is nothing I can do, and I changed the margins and font size of the inputs
